### DISCUSSION TITLE
Icons in ContextMenuHandler

### DISCUSSION THREAD

**Discussion by Daszbin** [cefsharp/CefSharp#4310] [Ideas]
So in all chromium based browsers icons are available in the context menu is this available in the Alloy runtime or is this only a Chrome runtime thing.

![](https://i.stack.imgur.com/adozn.jpg)

**Comment by amaitland** [comment]
To the best of my knowledge CEF doesn't currently support icons in its context menu implementation. You can implement your own custom context menu.
